File: 442. Find All Duplicates in an Array.swift
class Solution {
    func findDuplicates(_ nums: [Int]) -> [Int] {
        var rep: [Int] = []
        var num = nums
        num.sort()
        var i = 0
        while i < num.count - 1 {
            if num[i] == num[i + 1]{
                rep.append(num[i])
            }
            i += 1
        }
        return rep
    }
}

File: 442. Find All Duplicates in an Array.cs
public class Solution {
    public IList<int> FindDuplicates(int[] nums) {
        Array.Sort(nums);
        IList<int> disnum = new List<int>();  //using IList to get a list for there is in the function
        for (int i = 0; i < nums.Length - 1; i++) {
            if (nums[i] == nums[i + 1]) {
                disnum.Add(nums[i]);
            }
        }
        return disnum;
    }
}
